<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gzmapchangeperson</class>
 <widget class="QDialog" name="gzmapchangeperson">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>疫情地图管理系统</string>
  </property>
  <widget class="QLabel" name="UI_editer">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>432</width>
     <height>768</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/GUI/UI_editer.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="afterPerson">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>391</width>
     <height>65</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN Medium</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(49, 49, 49);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="maxLength">
    <number>8</number>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>249</x>
     <y>687</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/GUI/UI_btn_confirm_round.png);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>690</y>
     <width>141</width>
     <height>54</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/GUI/UI_btn_cancel_round.png);</string>
   </property>
  </widget>
  <widget class="QLabel" name="placeName">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>560</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="currentNum">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>399</y>
     <width>371</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN Bold</family>
     <pointsize>48</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(23, 174, 255);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel_old">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>30</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源宋体 CN Heavy</family>
     <pointsize>36</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>默认</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>30</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源宋体 CN Heavy</family>
     <pointsize>20</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>false</bool>
   </property>
   <item>
    <property name="text">
     <string>确诊人数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>境外输入人数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>境内输入人数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>本地输入人数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>天气</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>温度</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkNew">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>630</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>思源黑体 CN Medium</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string> 14天内新增</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
  </widget>
  <zorder>nameLabel_old</zorder>
  <zorder>placeName</zorder>
  <zorder>UI_editer</zorder>
  <zorder>afterPerson</zorder>
  <zorder>saveButton</zorder>
  <zorder>exitButton</zorder>
  <zorder>currentNum</zorder>
  <zorder>nameLabel</zorder>
  <zorder>checkNew</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
